dts/arm/st/l4/stm32l462Xe.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 Giancarlo Stasi
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/l4/stm32l462.dtsi>
+
+/ {
+	sram0: memory@20000000 {
+		reg = <0x20000000 DT_SIZE_K(160)>;
```

Comment:
SRAM is divided in 2 non contiguous zones (SRAM1 and SRAM2)
SRAM1 is 128Kb

Reply:
The datasheet (https://www.st.com/resource/en/datasheet/stm32l462ve.pdf), section 3.5, states:
" 32 Kbyte located at address 0x1000 0000 with hardware parity check (SRAM2).
This memory is also mapped at address 0x2002 0000, offering a contiguous address 
space with the SRAM1 (32 Kbyte aliased by bit band)"

It's same for L452 and L462.

Reply:
@giansta my bad I have missed this mapping. 160Kb it is then.